api/pom.xml

```diff
@@ -11,4 +11,19 @@
 	<packaging>jar</packaging>
 	<name>Basic Module API</name>
 	<description>API project for Basic Module</description>
+
+	<dependencies>
+        <dependency>
+            <groupId>org.openmrs.api</groupId>
```

Comment:
Was this dependency necessary to fix the reported error?

Reply:
@dkayiwa Yes, it was needed to import the class `org.openmrs.module.BaseModuleActivator` (to replace `org.openmrs.module.Activator` as it is deprecated)

Reply:
That is not correct. Changing to BaseModuleActivator does not require any dependency change.

Reply:
+1 @dkayiwa, please find a correction here: https://github.com/openmrs/openmrs-module-basicmodule/pull/4/commits/603765e604ee40add41dc29af238db840bf3272a